index.html

```diff
@@ -12,6 +12,9 @@
     <h2>{{supe.name}}:</h2>
     <h4>{{supe.number}}</h4>
     <h4>Affiliation: {{supe.group}}</h4>
+    <img ng-src="{{supe.img}}", ng-show="vm.viewImage[$index]">
+    <button ng-click="vm.viewImage[$index] = !vm.viewImage[$index]">Get Image</button>
+    <!-- ##this means that initially, the click value is evaluating to false, so ng-hide is applied to the image.  but then, by clicking the button, we toggle to the expression evaluating to true, which applies the show class## -->
```

Comment:
after getting hung up on the syntax here, I dug around the internet for a while and then ended up looking at what other people had submitted.
I think this is the logic, but could you look over what I've written here and let me know?  I want to make sure that I actually understand what I did:

i think the reason this works is because initially the ng-click expression evaluates to falsey, which is forcing the ng-show directive above it to switch over to ng-hide.  when we click on the button, the ng-click expression toggles over to true (because the act of clicking creates the opposite effect from whatever the starting point is), and an expression that evaluates to true is what causes ng-show to be called for the image.